### ISSUE TITLE
Enhancement: Drag and drop from UPK explorer into Package Ed for brushes

### ISSUE THREAD

**Issue by Kinkojiro** [ME3Tweaks/LegendaryExplorer#23]
Brushes are geometry contained entirely in unreal structs.  It would be good to be able to copy them from unreal (and therefore 3dsMax) into a pcc.  Ideally drag and drop from UDK Explorer into package ed, but if that is too difficult a simple import function would be useful.

Saves having to do it by hand/hexeditor

The following are brushes:  BrushComponent, RBBodySetup

BTW ME3Creator has a "Import function" that effectively has been surpassed by drag and drop.  However if dragging/dropping between UDK Explorer and Package Ed was difficult, it might be a useful interface for an importer.

**Comment by Mgamerz** [comment]
Can you give me some examples for UDK Explorer I can use? Such as an example file and destination pcc.

**Comment by Kinkojiro** [comment]
This .upk includes 2 collision brushes obj 3+4 - termed RB_BodySetup because they are static mesh collision brushes rather than volume brushes but the code is identical.

I want to replace the version in here: Obj 3750

[Contact me for files]

**Comment by Mgamerz** [comment]
Can you leave feedback on the new UDKPackage support in the toolset for what you need. Binary relinking is not yet done for staticmesh but will come eventually.

I personally have not tested if all properties are parsable - if you can export stuff from UDK that has things like boolproperties that would be helpful.

**Comment by Kinkojiro** [comment]
BodySetups seem to be fine.
Static Meshes appear to drag and drop incorrectly.  There must be some sorted of transformation/compression between UDK and ME.

**Comment by Mgamerz** [comment]
Not sure what you mean. I know the size changes because StrProperty in UDK is 1 byte for each character, while in ME3 it is 2 characters, so sizes will not line up.

**Comment by Kinkojiro** [comment]
Ok. Let me try again.

**Comment by Kinkojiro** [comment]
This is the problem.
(1) I import a staticmesh (EGM_SM_ObjectRho) via ME3Creator.  Imports correctly (meshplorer can view). object size is 483kb
(2) I import the same mesh again via drag and drop.  It copies in but is object size 4.8MB. Meshplorer throws an exception.
The files are on the chat.

**Comment by Mgamerz** [comment]
Just an update, this has been fixed, but I am not closing this issue because I want to add a "replace" option when drag/drop on an export of the same class (and maybe name).

**Comment by Mgamerz** [comment]
This has been implemented, but needs checked. From my limited tests replacing works, however I have not tested across games and UDK.

**Comment by Kinkojiro** [comment]
VALIDATED THANK YOU 3.1.2.14
